### PR TITLE
Create a payload for an APS hook to start APS Workflows

### PR DESCRIPTION
**Pull request by mmuller88**
Hi Axel. 

Thank you pretty much for this awesome action amp. Regarding to this tutorial https://hub.alfresco.com/t5/alfresco-process-services/using-rest-call-with-a-start-signal-event-in-aps/ba-p/288943 I managed to use the webhook to start an APS Flow with uploading to an ACS folder. Even better the document id is forwarded to the APS Workflow as well. Any feedback is much welcomed :).

If you allow I would like to write a blog post about my work + using your amp.

### CODE REVIEW

File: repository/src/main/messages/bootstrap.properties [hunk 1]
```diff
@@ -1,4 +1,5 @@
 ${moduleId}.spaces.templates.actionTemplates.name=Acosix Action Templates
 ${moduleId}.spaces.templates.webhookPayloadTemplates.name=Webhook Call Payload Templates
 ${moduleId}.content.templates.discordNewFilePayloadTemplate.title="New File" Discord Payload
-${moduleId}.content.templates.mattermostNewFilePayloadTemplate.title="New File" Mattermost Payload
\ No newline at end of file
+${moduleId}.content.templates.mattermostNewFilePayloadTemplate.title="New File" Mattermost Payload
+${moduleId}.content.templates.apsNewFilePayloadTemplate.title="New File" Aps Payload
```
Comment (AFaust) [inline repository/src/main/messages/bootstrap.properties]: Nit-picking here, but "Aps" should be "APS"

File: repository/src/main/config/bootstrap/content/aps_new_file.json.ftl [hunk 2]
```diff
@@ -0,0 +1,12 @@
...
+    "tenantId":"${tenantId!"tenant_1"}",
+    "async":"${async!"false"}",
+    "variables":
+    [
+        {
+            "name":"document",
+            "value":"${document.nodeRef}"
+        }
+    ]
+}</#escape>
```
Comment (AFaust) [inline repository/src/main/config/bootstrap/content/aps_new_file.json.ftl:12]: Just a general question (and maybe a bit of documentation for others): For which specific APS endpoint (URL) is this payload intended? I currently do not have access to APS source myself as my only APS customer has temporarily shut down work (and my access) due to Corona-related "Kurzarbeit".

**Review by AFaust — COMMENTED** [review]
No blockers for PR to be approved - just minor nit-picking and one question for better understanding.

File: repository/src/main/config/bootstrap/content/aps_new_file.json.ftl (hunk 2 above)
Comment (mmuller88) [reply repository/src/main/config/bootstrap/content/aps_new_file.json.ftl:12]: Hi Axel sorry for the late response. I think I didn't got notified in my email. I will try to figur ethat out. TBH I just took from the guide https://hub.alfresco.com/t5/alfresco-process-services/using-rest-call-with-a-start-signal-event-in-aps/ba-p/288943 I tried to let them out but then it didn't worked :)

Comment (mmuller88) [reply repository/src/main/config/bootstrap/content/aps_new_file.json.ftl:12]: If I understood correct APS is build on top of https://github.com/Activiti/Activiti . After digging around there a bit  (reading tests and look for keywords related to Signals) I think there is no specific signal endpoint as you have to create that yourself following the guide JTSmith. 

Anyway I think I found some related Definitions here :

https://github.com/Activiti/Activiti/blob/develop/activiti-core/activiti-engine/src/main/java/org/activiti/engine/impl/bpmn/behavior/IntermediateCatchSignalEventActivityBehavior.java

**Pull request merged**